### ISSUE TITLE
Site support request : http://lelombrik.net

### ISSUE THREAD

**Issue by skyghis** [rg3/youtube-dl#4534]
I tried to download some videos on this site and none worked. It download the _html_ page.
I found some different video formats. 
* http://lelombrik.net/67282 Youtube
* http://lelombrik.net/53136 Dailymotion
* http://lelombrik.net/67285 Vimeo
* http://lelombrik.net/67254 Hosted video ?

```
$youtube-dl -v -s http://lelombrik.net/67282 
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', '-s', 'http://lelombrik.net/67282']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2014.12.17.2
[debug] Python version 2.7.8 - Linux-3.16.0-28-generic-x86_64-with-Ubuntu-14.10-utopic
[debug] exe versions: none
[debug] Proxy map: {}
[generic] 67282: Requesting header
WARNING: Falling back on generic information extractor.
[generic] 67282: Downloading webpage
WARNING: URL could be a direct video link, returning it as such.
```

> Thanks for this great tool :+1:

**Comment by naglis** [comment]
This was actually due to a bug (see #4753) which has been fixed by 61ca9a8. The embedded (YouTube/vimeo/DailyMotion) and native lelombrik.net videos should be caught by the generic extractor in the next release of youtube-dl. Thanks for the report.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.01.23.2 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.

**Comment by skyghis** [comment]
I confirm the fix in version _2015.01.23.2_
Thanks
